### PR TITLE
added rpi setup instructions

### PR DESCRIPTION
**Pull request by arank**
updated NNVM makefile to remove x86 specific msse flag, and added Raspberry Pi build instructions for MxNet (waiting on amalgamation fixes to create build script)

### CODE REVIEW

File: docs/get_started/raspbian_setup.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Executing either of these commands creates a library called ```libmxnet.so```.
+
+### Install MXNet for Python On Raspberry Pi
+
+To install python bindings run the following commands:
+
+```bash
+cd setup-utils
+bash ./install-mxnet-ubuntu-python.sh
```
Comment (sandeep-krishnamurthy) [inline docs/get_started/raspbian_setup.md]: This script, installs MXNet along with all its dependencies (including open cv, open blas etc...). I guess you intended to run - setup.py script in python package?

File: docs/get_started/raspbian_setup.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Installing MXNet on Ubuntu
```
Comment (sandeep-krishnamurthy) [inline docs/get_started/raspbian_setup.md]: It says Ubuntu. May be Raspbian?

File: docs/get_started/raspbian_setup.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+MXNet currently supports the Raspbian operating system for Raspberry Pi Devices, offering a python API. We currently reccomend running MxNet on the Raspberry Pi 3 or other devices with more than 1GB of RAM and with an SD card that has at least 4 GB free.
+The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MxNet library (though they can run the mxnet amalgamation library). 
+
```
Comment (sandeep-krishnamurthy) [inline docs/get_started/raspbian_setup.md]: May be good idea to add a line about approx. memory usage by installing mxnet? This might give users an idea of what they are installing and what it takes on hardware

File: docs/get_started/raspbian_setup.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Clone the MXNet source code repository to your computer, using ```git```.
+```bash
+    git clone https://github.com/dmlc/mxnet.git --recursive
+```
+
+Then build the full MxNet library.
+```bash
+    cd mxnet
+    export ARM=1
```
Comment (sandeep-krishnamurthy) [inline docs/get_started/raspbian_setup.md]: A comment explaining why is this needed or how it will be used by mxnet.

File: docs/get_started/raspbian_setup.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+### Install MXNet for Python On Raspberry Pi
+
+To install python bindings run the following commands:
+
+```bash
+cd setup-utils
+bash ./install-mxnet-ubuntu-python.sh
+```
+
+Sometimes this you have to run the bash command twice as it may fail the first time.
```
Comment (sandeep-krishnamurthy) [inline docs/get_started/raspbian_setup.md]: Why?

File: docs/get_started/raspbian_setup.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+You are now ready to run MxNet on your Raspberry Pi! 
+
+**Note - ** Keep in mind loading the entire MxNet library takes up a significant fraction of RAM, so steps such as switching off the GUI to free RAM may be nessecary to load training data or large models into memory.
```
Comment (sandeep-krishnamurthy) [inline docs/get_started/raspbian_setup.md]: Approx RAM consumption would be ideal to let users know about this Note.

File: docs/get_started/raspbian_setup.md (hunk 1 above)
Comment (arank) [inline docs/get_started/raspbian_setup.md]: If MxNet is already installed though it just calls setup.py and checks the dependencies are met.

File: docs/get_started/raspbian_setup.md (hunk 5 above)
Comment (arank) [inline docs/get_started/raspbian_setup.md]: No clue. This was a weird error I kept encountering

File: docs/tutorials/embedded/wine_detector.md [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+python camera_test.py
+```
+
+If this is working you see a preview every few seconds of the image that is being captured and fed to the model, as well as predicted classes for objects in the image being written to the terminal. 
+
+Try pointing the PiCamera at a few different objects and see the predictions the network comes out with.
+
+## Connecting Our Model To The AWS Cloud
+
+We can now move on to adding the code to send the predictions this real time model is making locally to the AWS cloud if certain conditions are met.
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: real time -> real-time

Add a word:
predictions **that** this

File: docs/tutorials/embedded/wine_detector.md [hunk 8]
```diff
@@ -0,0 +1,386 @@
...
+>>> import inception_predict
+>>> predict_from_url("http://imgur.com/HzafyBA")
+```
+
+This should give a reasonable prediction for the fuzzy cow in this [image](http://imgur.com/HzafyBA). 
+
+
+## Running a Inception on Real-Time Video From PiCamera
+
+We can now move on to using this network for object detection in real time video from the PiCamera.
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: real time -> real-time

File: docs/tutorials/embedded/wine_detector.md [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+
+To follow this tutorial, you must setup your Pi as instructed (preferably from a fresh Raspbian install), then create the files and run the bash commands described below. All instuctions described are can be executed on the Raspberry Pi directly or via SSH.
+
+<!--A bulleted list of the tasks the reader will accomplish and skills he or she will learn. Begin each list item with a noun (Learn, Create, Use, etc.). -->
+
+You will accomplish the following:
+
+- Build and Install MxNet with Python bindings on your Raspbian Based Raspberry Pi
+- Fetch and run a pre-trained MxNet model on your Pi
+- Create a real time video analysis application for the Pi
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: real time -> real-time

File: docs/tutorials/embedded/wine_detector.md [hunk 10]
```diff
@@ -0,0 +1,386 @@
+
+# Real Time Object Detection with MxNet On The Raspberry Pi
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: Real Time -> Real-time

File: docs/tutorials/index.md [hunk 11]
```diff
@@ -115,6 +115,13 @@ The following tutorials show how visualization helps us in daily work, and bette
 - [Understanding the vanishing gradient problem through visualization](https://github.com/dmlc/mxnet-notebooks/blob/master/python/tutorials/understanding_vanish_gradient.ipynb)
 *An example of using the visualization component, [TensorBoard](https://github.com/dmlc/tensorboard), to have an intuitive understanding on the well-known vanished gradient problem in deep learning.*
 
+#### Embedded
+
+The following tutorials explain how to develop machine learning applications running on embedded devices, such as the Raspberry Pi
+
+- [Raspberry Pi Object Classifier](http://mxnet.io/tutorials/embedded/wine_detector.html)
+*An example using a Raspberry Pi equipped with the standard camera module to identify objects in real time using a pretrained imageNet model*
```
Comment (lmatuszek) [inline docs/tutorials/index.md]: real time -> real-time

File: docs/tutorials/embedded/wine_detector.md [hunk 12]
```diff
@@ -0,0 +1,386 @@
...
+    camera.stop_preview()
+    topn = inception_predict.predict_from_local_file(filename, N=5)
+    
+    # Check if either of the top two predictions are wine related and publish a message if it is
+    # you can change 'wine' here to anything you want to alert the server about detecting
+    if 'wine' in topn[0][1] or 'wine' in topn[1][1]: 
+        myAWSIoTMQTTClient.publish("sdk/test/Python", "New Message: WINE DETECTED!", 0)
+```
+
+You can then run this file by entring the following command
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: entring -> entering

File: docs/tutorials/embedded/wine_detector.md [hunk 13]
```diff
@@ -0,0 +1,386 @@
...
+
+```bash
+python wine_alerter.py -e <endpointURL> -r <rootCAFilePath> -c <certFilePath> -k <privateKeyFilePath>
+```
+
+If this is working you should see the same kind of image preview you did with camera_test.py every few seconds, however the console will only print a message now when a wine bottle is detected in the shot (you can edit the bottom lines in the wine_alerter.py code to make this alert for any object you specify).
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: We might want to call out that this only works for words in the ImageNet dataset.

File: docs/tutorials/embedded/wine_detector.md [hunk 14]
```diff
@@ -0,0 +1,386 @@
...
+
+You can open up the activity tab for the thing you registered on the AWS IoT Dashboard and see the corresponding messages pushed to the server whenever a wine bottle is detected in shot. Thanks to AWS IoT even if network connectivity periodically fails, the updates will eventually be pushed out to the server when possible, allowing this system to reliably let you know when there is wine around.
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: Suggested word tweaking:
You can open up the activity tab for the thing **that** you registered on the AWS IoT Dashboard and see the corresponding messages pushed to the server whenever a wine bottle is detected in **a camera** shot. **Even if network connectivity periodically fails, AWS IoT will push updates out to the server when possible,** allowing this system to reliably let you know when there is wine around.

File: docs/tutorials/embedded/wine_detector.md [hunk 15]
```diff
@@ -0,0 +1,386 @@
...
+<!-- A brief introduction to the tutorial that describes:
+
+- The problem that that the tutorial addresses
+- Who the intended audience is
+- The expected experience level of that audience with a concept or tool 
+- Which environment/language/hardware it runs in 
+
+If there is another similar tutorial that's more appropriate for another audience, direct the reader there with a linked reference. -->
+
+### What's In This Tutorial?
```
Comment (lmatuszek) [inline docs/tutorials/embedded/wine_detector.md]: Why is this level three? Shouldn't it be level two, as it's right after "Real Time Object Detection with MxNet On The Raspberry Pi "?

File: docs/get_started/raspbian_setup.md [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+## Standard installation
+
+Installing MXNet is a two-step process:
+
+1. Build the shared library from the MXNet C++ source code.
+2. Install the supported language-specific packages for MXNet.
+
+### Build the Shared Library
+
+On Rasbian versions Wheezy and later, you need the following dependencies:
```
Comment (arank) [inline docs/get_started/raspbian_setup.md]: I have not tested with earlier versions, but from my past experience with Pis you will likely need a significant amount of updates to packages etc and it is easier to just assume a common base distribution to work from.

File: docs/get_started/raspbian_setup.md [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+### Install MXNet for Python On Raspberry Pi
+
+To install python bindings run the following commands:
+
+```bash
+cd setup-utils
+bash ./install-mxnet-ubuntu-python.sh
+```
+
+Sometimes you have to run the bash command twice as it may fail the first time.
```
Comment (arank) [inline docs/get_started/raspbian_setup.md]: This script fully builds and installs MXNet for x86. Since I had to follow a custom process to compile for ARM this script is basically updating dependencies (where it seems to be failing), checking the mxnet binary exists and running "python setup.py install". I think we can just change this to cd mxnet/python; sudo python setup.py install but I'll test it out.

File: docs/tutorials/embedded/wine_detector.md [hunk 18]
```diff
@@ -0,0 +1,386 @@
...
+* An AWS account With AWS IoT enabled and the [AWS IoT Python SDK](https://github.com/aws/aws-iot-device-sdk-python) (for remote, real-time managing and monitoring of the model running on the Pi)
+* The [cv2 Python library](http://www.pyimagesearch.com/2015/02/23/install-opencv-and-python-on-your-raspberry-pi-2-and-b/) for the Pi
+
+## Building MxNet for The Pi
+
+The first step will be to get MxNet with the Python bindings running on your Raspberry Pi 3. There is a tutorial for that provided on [here](http://mxnet.io/get_started/raspbian_setup.html). In short you will have to download the dependencies, and build the full MxNet library for the Pi with the ARM specific compile flags. Be sure to build the library with open CV as we will be using a model that requires it to process images. Then you will finally the Python bindings. Once this is done you should test that works by opening a python REPL on your Pi and typing the following commands:
```
Comment (arank) [inline docs/tutorials/embedded/wine_detector.md]: I think register would work better here

File: docs/tutorials/embedded/wine_detector.md [hunk 19]
```diff
@@ -0,0 +1,386 @@
...
+# Load the network parameters
+sym, arg_params, aux_params = mx.model.load_checkpoint('Inception_BN', 0)
+
+# Load the network into an MxNet module and bind the corresponding parameters
+mod = mx.mod.Module(symbol=sym, context=mx.cpu())
+mod.bind(for_training=False, data_shapes=[('data', (1,3,224,224))])
+mod.set_params(arg_params, aux_params)
+ 
+'''
+Function to run a forward pass through the model given a pointer to an image file and the loaded network
```
Comment (arank) [inline docs/tutorials/embedded/wine_detector.md]: I think "model and loaded network" is a bit redundant but I like the re-statement

File: docs/tutorials/embedded/wine_detector.md [hunk 20]
```diff
@@ -0,0 +1,386 @@
...
+
+Now that we have defined inception_predict.py we can test that the model is running correctly. Open a Python REPL in your home directory and enter the following:
+
+```bash
+python
+>>> import inception_predict
+>>> predict_from_url("http://imgur.com/HzafyBA")
+```
+
+This should give a reasonable prediction for the fuzzy cow in this [image](http://imgur.com/HzafyBA).
```
Comment (arank) [inline docs/tutorials/embedded/wine_detector.md]: You are totally right! It is definitely fluffy 👍

File: docs/get_started/raspbian_setup.md [hunk 21]
```diff
@@ -1,7 +1,7 @@
 # Installing MXNet on Raspbian
-MXNet currently supports the Debian based Raspbian operating system for Raspberry Pi Devices, offering a python API. 
+MXNet currently supports a python API for the Debian based Raspbian operating system for Raspberry Pi Devices.
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: MXNet supports a Python API for the Debian-based Raspbian operating system so you can run MXNet on Raspberry Pi devices.

ED: This is an abrupt intro. Installation should be a step in the tutorial, not a separate topic.

File: docs/get_started/raspbian_setup.md [hunk 22]
```diff
@@ -1,7 +1,7 @@
...
 
-The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to this we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library). 
+The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to the size we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library).
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: The complete MXNet library exceeds 200 MB and the requirements can take almost 1 GB of disk space. Because of this, we recommend running MXNet on the Raspberry Pi 3 or an equivalent device that has more than 1 GB of RAM and a Secure Digital (SD) card that has at least 4 GB of free memory. The Raspberry Pi 1, 2, Zero and other devices with less than 1 GB of RAM cannot run the compete MXNet library.

File: docs/get_started/raspbian_setup.md [hunk 23]
```diff
@@ -1,7 +1,7 @@
...
+MXNet currently supports a python API for the Debian based Raspbian operating system for Raspberry Pi Devices.
 
-The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to this we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library). 
+The full MxNet library is over 200MB when loaded into memory and the requirements can take almost 1GB of disk space. Due to the size we currently reccomend running MXNet on the Raspberry Pi 3 or equivilant devices with more than 1GB of RAM and with an SD card that has at least 4 GB of memory free. The Raspberry Pi 1, 2, Zero and other devices with less than 1GB of RAM are not sufficient to run the full MXNet library (though they can run the MXNet amalgamation library). 
 
 ## Standard installation
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: Standard Installation

File: docs/get_started/raspbian_setup.md [hunk 24]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
     git clone https://github.com/dmlc/mxnet.git --recursive
 ```
 
-Then build the full MxNet library, ensuring the ARM flag is enabled (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the compiled library. You can do this by compiling with the following code:
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: If you aren't processing images with MXNet on the Pi, you can minimize the size of the compiled library by building MXNet without the Open Source Computer Vision (OpenCV) library with the following code:

File: docs/get_started/raspbian_setup.md [hunk 25]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
 ```bash
     cd mxnet
     export ARM=1
+    export USE_OPENCV = 0
     make
 ```
 
-If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the loaded library. You can do this by compiling with the following code:
+Otherwise you can build the full MxNet library:
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: Otherwise, you can build the complete MXNet library:

File: docs/get_started/raspbian_setup.md [hunk 26]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
-If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the loaded library. You can do this by compiling with the following code:
+Otherwise you can build the full MxNet library: 
 ```bash
     cd mxnet
     export ARM=1
-    export USE_OPENCV = 0
     make
 ```
 
+Whenever you are compiling MXNet for the Pi you need to ensure the "ARM" environment variable is set to 1 (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: For ARM compatibility, set the ARM environment variable to 1. This removes the x86-specific -msse flag from the NNVM submodule's makefile.

File: docs/get_started/raspbian_setup.md [hunk 27]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
+
 Executing either of these commands creates a library called ```libmxnet.so```.
 
-### Install MXNet for Python On Raspberry Pi
+## Install MXNet for Python On Raspberry Pi
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: Install MXNet for Python on Raspberry Pi

File: docs/get_started/raspbian_setup.md [hunk 28]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
 
 To install python bindings run the following commands:
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: To install Python bindings, run the following commands:

File: docs/get_started/raspbian_setup.md [hunk 29]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
 
 ```bash
-cd setup-utils
-bash ./install-mxnet-ubuntu-python.sh
+    cd python
+    sudo python setup.py install
 ```
 
-Sometimes you have to run the bash command twice as it may fail the first time.
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: If the Bash command fails the first time, run it again.

File: docs/get_started/raspbian_setup.md [hunk 30]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
-
 You are now ready to run MxNet on your Raspberry Pi!
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: You are now ready to run MXNet on your Raspberry Pi device.

File: docs/get_started/raspbian_setup.md [hunk 31]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
+    sudo python setup.py install
 ```
 
-Sometimes you have to run the bash command twice as it may fail the first time.
-
 You are now ready to run MxNet on your Raspberry Pi! 
 
-*Note - Keep in mind loading the entire MxNet library takes up a significant fraction (200MB+) of the Raspberry Pi's limited RAM, so steps such as switching off the GUI and killing concurrently running processes to free RAM may be nessecary to load training data or large models into memory.*
-
+*Note - Keep in mind that loading the entire MXNet library takes up a significant fraction (200MB+) of the Raspberry Pi's limited RAM, so you might have to switch off the GUI and kill any concurrently running processes to free the RAM needed to load training data or large models into memory.*
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: ***Note***
Remember that the entire MXNet library takes up a significant amount (200 MB or more) of the Raspberry Pi's limited RAM. To load training data or large models into memory, you might have to turn off the GUI and terminate running processes to free the RAM.

File: docs/get_started/raspbian_setup.md [hunk 32]
```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
...
 
 ## Next Steps
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: ED: If you aren't providing links to other topics, remove this heading.

File: docs/tutorials/embedded/wine_detector.md [hunk 33]
```diff
@@ -1,5 +1,5 @@
 
-# Real Time Object Detection with MxNet On The Raspberry Pi  
+# Real-time Object Detection with MxNet On The Raspberry Pi
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: Real-time Object Detection with MxNet on a Raspberry Pi

File: docs/tutorials/embedded/wine_detector.md [hunk 34]
```diff
@@ -10,30 +10,29 @@
 
 If there is another similar tutorial that's more appropriate for another audience, direct the reader there with a linked reference. -->
 
-### What's In This Tutorial?
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: What's in This Tutorial?

File: docs/tutorials/embedded/wine_detector.md [hunk 35]
```diff
@@ -10,30 +10,29 @@
...
-In this demo we will walk through the steps to setup a real-time object classifier, powered by MxNet running entirely on a Raspberry Pi 3. We will then work through connecting this local Raspberry Pi model to the AWS cloud via the IoT service, letting you easily and reliablely get realtime updates from the device.
+## What's In This Tutorial?
+In this demo we will walk through the steps to set up a real-time object classifier, powered by MxNet running entirely on a Raspberry Pi 3. We will then work through connecting this local Raspberry Pi model to the AWS cloud via the IoT service, letting you easily and reliably get real-time updates from the device.
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: +This tutorial shows how to:

1. Use MXNet to set up a real-time object classifier on a Raspberry Pi 3 device. 

2. Connect the local Raspberry Pi model to the AWS Cloud with AWS IoT to get real-time updates from the device.

File: docs/tutorials/embedded/wine_detector.md [hunk 36]
```diff
@@ -10,30 +10,29 @@
...
 
-* The primary goal of this Tutorial is to introduce developers who work with the Raspberry Pi or similar embedded ARM-based devices with a guide to compiling MxNet for those devices as well as running a non-trival, pre-trained deep network model on them. 
+* The primary goal of this tutorial is to provide developers who work with the Raspberry Pi or similar embedded ARM-based devices with a guide to compiling MxNet for those devices, as well as running a non-trivial, pre-trained deep network model.
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: This tutorial shows developers who work with the Raspberry Pi or similar embedded ARM-based devices how to compile MXNet for those devices and running a pretrained deep network model.

File: docs/tutorials/embedded/wine_detector.md [hunk 37]
```diff
@@ -10,30 +10,29 @@
...
 
-* The secondary goal of this Tutorial is to provide embedded device developers with an example of how AWS IoT infrastructure can be used to manage and monitor these MxNet models once they are running across your devices.
+* The secondary goal of this Tutorial is to provide embedded device developers with an example of how AWS IoT infrastructure can be used to manage and monitor these MxNet models once they are running on your devices.
 
 ### Who's This Tutorial For?
 This tutorial is aimed at developers who are reasonably familiar with the Raspbian OS and the [Raspberry Pi Ecosystem](https://www.raspberrypi.org/), as well as somewhat familiar with machine learning, MxNet, and [AWS IoT](https://aws.amazon.com/iot/). All code written will be in Python 2.7
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: This tutorial is written for developers who are familiar with the Raspbian operating system and the [Raspberry Pi ecosystem](https://www.raspberrypi.org/) and are somewhat familiar with machine learning, MXNet, and [AWS IoT](https://aws.amazon.com/iot/). All code is written in Python 2.7.

File: docs/get_started/raspbian_setup.md (hunk 22 above)
Comment (arank) [inline docs/get_started/raspbian_setup.md]: I made a typo here and said disk space when I meant RAM

File: docs/get_started/raspbian_setup.md (hunk 24 above)
Comment (arank) [inline docs/get_started/raspbian_setup.md]: A little pedantic but 'code' should actually be 'command' in the context of bash

File: docs/get_started/raspbian_setup.md (hunk 32 above)
Comment (arank) [inline docs/get_started/raspbian_setup.md]: Just going to link to the tutorials section for embedded devices.

File: docs/get_started/raspbian_setup.md [hunk 38]
```diff
@@ -24,52 +28,47 @@ On Rasbian versions Wheezy and later, you need the following dependencies:
...
-    cd mxnet
-    export ARM=1
     make
 ```
 
-Whenever you are compiling MXNet for the Pi you need to ensure the "ARM" environment variable is set to 1 (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+Executing either of these commands creates a file called ```libmxnet.so``` in the mxnet/lib directory.
 
-Executing either of these commands creates a library called ```libmxnet.so```.
+*Note - If you are getting build errors it is likely you are on an older version of MXNet, and there are x86 specific -msse CFLAGS hardcoded in the project Makefile or one of the submodule Makefiles that need to be manually removed.*
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: **Note**
 If you are getting build errors, it is likely you are using an older version of MXNet and there are x86-specific -msse CFLAGS hardcoded in the project makefile or one of the submodule makefiles. Manually remove them.

File: docs/get_started/raspbian_setup.md [hunk 39]
```diff
@@ -24,52 +28,47 @@ On Rasbian versions Wheezy and later, you need the following dependencies:
...
 ```bash
     cd python
     sudo python setup.py install
 ```
 
-You are now ready to run MxNet on your Raspberry Pi! 
+You are now ready to run MXNet on your Raspberry Pi device.
 
-*Note - Keep in mind that loading the entire MXNet library takes up a significant fraction (200MB+) of the Raspberry Pi's limited RAM, so you might have to switch off the GUI and kill any concurrently running processes to free the RAM needed to load training data or large models into memory.*
+*Note - Because the complete MXNet library takes up a significant amount of the Raspberry Pi's limited RAM, when loading training data or large models into memory, you might have to turn off the GUI and terminate running processes to free RAM.*
```
Comment (annbech) [inline docs/get_started/raspbian_setup.md]: **Note**
Because the complete MXNet library takes up a significant amount of the Raspberry Pi's limited RAM, when loading training data or large models into memory, you might have to turn off the GUI and terminate running processes to free RAM.

File: docs/tutorials/embedded/wine_detector.md [hunk 40]
```diff
@@ -1,6 +1,7 @@
 
 # Real-time Object Detection with MxNet On The Raspberry Pi
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: Real-time Object Detection with MxNet on the Raspberry Pi

File: docs/tutorials/embedded/wine_detector.md [hunk 41]
```diff
@@ -11,14 +12,16 @@
 If there is another similar tutorial that's more appropriate for another audience, direct the reader there with a linked reference. -->
 
 ## What's In This Tutorial?
```
Comment (annbech) [inline docs/tutorials/embedded/wine_detector.md]: ## What's in This Tutorial?

File: docs/tutorials/index.md [hunk 42]
```diff
@@ -120,7 +120,7 @@ The following tutorials show how visualization helps us in daily work, and bette
 The following tutorials explain how to develop machine learning applications running on embedded devices, such as the Raspberry Pi
```
Comment (annbech) [inline docs/tutorials/index.md]: The following tutorials explain how to develop machine learning applications running on embedded devices, such as the Raspberry Pi:

File: docs/tutorials/index.md [hunk 43]
```diff
@@ -120,7 +120,7 @@ The following tutorials show how visualization helps us in daily work, and bette
...
 
 - [Raspberry Pi Object Classifier](http://mxnet.io/tutorials/embedded/wine_detector.html)
-*An example using a Raspberry Pi equipped with the standard camera module to identify objects in real time using a pretrained imageNet model*
+*An example using a Raspberry Pi equipped with the standard camera module to identify objects in real-time using a pretrained imageNet model*
```
Comment (annbech) [inline docs/tutorials/index.md]: An example using a Raspberry Pi equipped with the standard camera module to identify objects in real-time using a pretrained imageNet model.
